### PR TITLE
Remove references to old edge vscode extension

### LINKED ISSUE #1032: "start" is not a linux/macOS command.
This document is written for Windows and does not include commands for Linux and MacOS.  For example, "start" is a Windows only command.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c648fd42-706c-bb78-2f32-ff6ace125551
* Version Independent ID: 68efb0c6-9a4d-e6f6-4639-c0a9faa4fa39
* Content: [Debug Microsoft Edge (Chromium) from Visual Studio Code - Microsoft Edge Development](https://docs.microsoft.com/en-us/microsoft-edge/visual-studio-code/debugger-for-edge)
* Content Source: [microsoft-edge/visual-studio-code/debugger-for-edge.md](https://github.com/MicrosoftDocs/edge-developer/blob/master/microsoft-edge/visual-studio-code/debugger-for-edge.md)
* Product: **microsoft-edge**
* GitHub Login: @MSEdgeTeam
* Microsoft Alias: **msedgedevrel**

### PR DESCRIPTION
**Pull request by captainbrosset**
VSCode now has a built-in browser debugger, but our docs where still
mentioning old information about installing an extension to debug edge.
VSCode also has really good documentation about how to configure the
debugger, so there really isn't a need for us to re-do our own
documentation but should instead link to the existing one.

Fixes #1032.

### CODE REVIEW

**Comment by captainbrosset** [comment]
@codepo8 please take a look at this when you get a chance. It seems like our docs about vscode were a bit outdated. 

The index page was mentioning 3 extensions: the debugger for edge, the edge tools, and webhint.
While the last 2 still exist and are definitely things we want to document, the first one is deprecated as vscode has a built-in browser debugger. 
So I removed references to this on the index page.

And I removed most of the docs from the "debug edge" page and instead linked to https://code.visualstudio.com/docs/nodejs/browser-debugging

What do you think?

**Review by Reezaali — DISMISSED** [review]

**Comment by captainbrosset** [comment]
@codepo8 mentioned elsewhere that there was already a PR to update the vs code docs. I think I found it: https://github.com/MicrosoftDocs/edge-developer/pull/1053 
Waiting for guidance from Chris on what is the right move before going forward with this PR.

File: microsoft-edge/visual-studio-code/debugger-for-edge.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-description: How to debug Microsoft Edge (Chromium) and Microsoft Edge (EdgeHTML) from Visual Studio Code
+description: How to debug Microsoft Edge (Chromium) from Visual Studio Code
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/debugger-for-edge.md:2]: update the date: field

File: microsoft-edge/visual-studio-code/index.md [hunk 2]
```diff
@@ -20,17 +20,11 @@ To acquire any of the extensions highlighted below, navigate to Extensions \(sel
 
 Search the Marketplace for the specific extension and choose **Install**.
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/index.md:21]: update the date: field

Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/index.md:21]: change:
navigate to Extensions
to:
navigate to **Extensions**
since it's a UI item to select.

Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/index.md:21]:
```suggestion
Search the Marketplace for the specific extension and select **Install**.  
```
per https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui

Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/index.md:21]: Change:
lightweight, but powerful 
to:
lightweight but powerful

File: microsoft-edge/visual-studio-code/index.md [hunk 3]
```diff
@@ -20,17 +20,11 @@ To acquire any of the extensions highlighted below, navigate to Extensions \(sel
 
 Search the Marketplace for the specific extension and choose **Install**.  
 
-:::image type="complex" source="./media/vscode-debugger-install.png" alt-text="Install the Debugger for Microsoft Edge Visual Studio Code extension" lightbox="./media/vscode-debugger-install.png":::
-   Install the **Debugger for Microsoft Edge** Visual Studio Code extension  
+:::image type="complex" source="./media/vscode-edge-tools-install.png" alt-text="Install the Microsoft Edge Tools for Visual Studio Code extension" lightbox="./media/vscode-edge-tools-install.png":::
+   Install the **Microsoft Edge Tools** Visual Studio Code extension
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/index.md:24]: Change:
Hints appear as inline underlines and are summarized
to:
Hints appear as wavy underlines in the text editor, and are summarized

**Review by mikehoffms — CHANGES_REQUESTED** [review]
Did an Edit pass.

**Comment by captainbrosset** [comment]
Thanks @mikehoffms for the review. I'll push a couple of new commits with the corresponding fixes.

**Review by mikehoffms — APPROVED** [review]

**Comment by mikehoffms** [comment]
> @codepo8 please take a look at this when you get a chance. It seems like our docs about vscode were a bit outdated.
> 
> The index page was mentioning 3 extensions: the debugger for edge, the edge tools, and webhint.
> While the last 2 still exist and are definitely things we want to document, the first one is deprecated as vscode has a built-in browser debugger.
> So I removed references to this on the index page.
> 
> And I removed most of the docs from the "debug edge" page and instead linked to https://code.visualstudio.com/docs/nodejs/browser-debugging
> 
> What do you think?

@codepo8 - could you please review & approve this PR?

**Review by codepo8 — APPROVED** [review]
This looks good to me, let's remove the content. #sign_off

**Pull request merged**